pkg/operator/ceph/object/user.go

```diff
@@ -149,35 +151,6 @@ func CreateUser(c *Context, user ObjectUser) (*ObjectUser, int, error) {
 	return decodeUser(result)
 }
 
-// UpdateUser updates the user whose ID matches the user.
-func UpdateUser(c *Context, user ObjectUser) (*ObjectUser, int, error) {
```

Comment:
@leseb : Why this api is removed? I agree currently this API is not used anywhere. But in future we might need this API. There was a request from an upstream user to support quota for `cephobjectstoreuser`. In those cases this API will helpful

Reply:
I was thinking we could use `ModifyUser()` from the API when the time comes. There is no need for `radosgw-admin` commands anymore.

Reply:
Okay I agree. Please don't forget to add it back